### PR TITLE
QA : How to manage Issues

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | QA : How to manage Issues
| Fixed ticket | N/A

### CODE REVIEW

**Comment by Progi1984** [comment]
Ping @Robin-Fischer-PS & @boubkerbribri

File: content/project-organization/quality-council/processes/issue-management.md [hunk 1]
```diff
@@ -10,107 +10,122 @@ This document details the processes in place for the position of "Issues Manager
 
 ## The Issue Manager job
 
-Your job is to **qualify issues** submitted to the GitHub project. Good issues help us discover bugs and provide ideas to improve PrestaShop and the native modules. These are the three questions you must ask yourself:
+Your job is to **qualify issues** submitted to the GitHub project. Good issues help us discover bugs and provide ideas to improve PrestaShop and the native modules.These are the tree questions you must ask yourself:
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/issue-management.md:13]: really? 🌴

File: content/project-organization/quality-council/processes/issue-management.md [hunk 2]
```diff
@@ -10,107 +10,122 @@ This document details the processes in place for the position of "Issues Manager
...
-- **Don't be a robot either.** Don't take these guidelines to the letter. If the reporter makes an effort, try to be flexible.
+- **Don't provide free support.** Your job isn't to help the reporter fix their problem, but to determine if the problem is caused by a defect in the software (or in documentation).
+- **Be pragmatic about your efforts.** It's the reporter's responsibility to provide the information needed to reproduce the problem in an understandable way. Don't waste your energy doing guesswork or waiting for a response.
+- **Don't be a robot either.** Don't take these guidelines to the letter. If the reporter makes an effort, try to be flexible.
 
 ## Issue state & labels
 
-GitHub has only two states for issues: open and closed. We use labels to keep track of issue type, intermediate state, severity and resolution.
+Github has only two states for issues: open and closed. We use labels to keep track of issue type, intermediate state, severity and resolution.
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/issue-management.md:27]: `GitHub` is correct for me.

File: content/project-organization/quality-council/processes/issue-management.md [hunk 3]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
 
 1. Answer using the "Duplicate Issue" template.
 2. Add the `Invalid` and `Duplicate` labels.
 3. Close the issue.
 
-#### Reproducing the issue
+### Reproducing the issue
 
-We must be able to reproduce the bug in a controlled environment. You can record your attempt, use a screen recorder (like [Screencastify](https://www.screencastify.com/), a Chrome extension), download it and then upload it on GitHub, post it in a comment so that the OP can see the steps you used to reproduce their issue.
+We must be able to reproduce the bug in a controlled environment. You can record your attempt, use a screen recorder (like [Screencastify](https://www.screencastify.com/), a Chrome extension), download it and then upload it on Github, post it in a comment so that the issue’s author can see the steps you used to reproduce their issue, either on the current branch or on develop branch, depending of what the author declared in his issue.
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/issue-management.md:189]: `GitHub` (or please fix the whole file)

File: content/project-organization/quality-council/processes/issue-management.md [hunk 4]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
 
-##### If the issue cannot be reproduced
+#### If t**he issue cannot be reproduced**
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/issue-management.md:191]: wrong format with `** ... **`

File: content/project-organization/quality-council/processes/issue-management.md [hunk 5]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
+Note that the product team must be able to understand the bug to be able to classify it in an EPIC. This means that if the issue is not correctly described nor labelled, or if this is a very tech issue, it MUST NOT be added to the kanban.
 
-#### Rewriting the issue
+### Rewriting the issue
 
-Once all the steps have been completed, the content of the issue should be rewritten if it is unclear, or if the author has clarified in the comments. All necessary information should be available in the body of the issue! It may be necessary **to rewrite the title if it is too generic** (e.g. "Help! BUG!") or if it does not accurately reflect the bug. Pay special attention to the **reproduction steps** ("how to reproduce")!
+Once all the steps have been completed, the content of the issue should be rewritten if it is unclear, or if the author has clarified in the comments. All necessary information should be available in the body of the issue!It may be necessary to rewrite the title if it is too generic (e.g. "Help! BUG!") or if it does not accurately reflect the bug.Pay special attention to the **reproduction steps** ("how to reproduce")!
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/issue-management.md:255]: `issue! It may be necessary`

missing space

File: content/project-organization/quality-council/processes/issue-management.md [hunk 6]
```diff
@@ -325,64 +343,63 @@ Since we had no news from you for more than 20 days, I'll close this ticket to a
...
 ```
 
-###### Regression
+#### Regression
 
 If it’s a regression, add this sentence:
 
 ```
-I didn't manage to reproduce the issue with PS x.x.x, I only reproduce it with PS x.x.x. So it's a regression!
+
```
Comment (nicosomb) [inline content/project-organization/quality-council/processes/issue-management.md:379]: why did you remove the line?

**Review by nicosomb — CHANGES_REQUESTED** [review]

File: content/project-organization/quality-council/processes/issue-management.md (hunk 1 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:13]: Fixed

File: content/project-organization/quality-council/processes/issue-management.md (hunk 3 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:189]: Fixed

File: content/project-organization/quality-council/processes/issue-management.md (hunk 4 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:191]: Fixed

File: content/project-organization/quality-council/processes/issue-management.md (hunk 5 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:255]: Thanks

File: content/project-organization/quality-council/processes/issue-management.md (hunk 6 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:379]: There is no more regression.

**Comment by Progi1984** [comment]
Ping @Robin-Fischer-PS & @boubkerbribri

**Comment by Progi1984** [comment]
@PrestaShop/prestashop-core-developers Could you remove the label, please ?

**Review by nicosomb — APPROVED** [review]

File: content/project-organization/quality-council/processes/issue-management.md [hunk 7]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
+- *Deprecated - Regression or not (since August 2023),*
+- `Verified` (if we reproduce the issue)
+- Category (`FO`/`BO`),
+- Affected page or concerned feature (ex: `Order`, `Module`, …),
 - Affected version(s),
 - Status (NMI, Needs Specs, Ready...)
 - ... (non-exhaustive list)
 
-A list of labels and their use is [available here](https://github.com/PrestaShop/PrestaShop/labels).
+A list of labels and their use is [available here](https://docs.google.com/spreadsheets/d/118iqWIkW24Rwn6eY8_7FnPhAacGXwA2RdGPzR_LXvy4/edit#gid=0).
```
Comment (kpodemski) [inline content/project-organization/quality-council/processes/issue-management.md:239]: @Progi1984 not available publicly

File: content/project-organization/quality-council/processes/issue-management.md [hunk 8]
```diff
@@ -325,16 +343,16 @@ Since we had no news from you for more than 20 days, I'll close this ticket to a
...
 ```
 Hello,
 
 I reproduce the issue with PrestaShop version x.x.x, I'll add this to the backlog so it can be fixed.
 
-Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project's own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you.
+Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you. 
 
-If you fix the issue on your end, please [contribute it back to the project]({{< devdocs "contribute/contribute-pull-requests/" >}}). Remember that the more people contribute, the better PrestaShop becomes for everyone.
+If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop-project.org/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.
```
Comment (kpodemski) [inline content/project-organization/quality-council/processes/issue-management.md:355]:
```suggestion
If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop-project.org/8/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.
```

although the `devdocs` helper should've worked 🤔

File: content/project-organization/quality-council/processes/issue-management.md [hunk 9]
```diff
@@ -344,45 +362,36 @@ You can personalize the message according to your needs, for example:
...
-
-Read more about how we expect the issues to be handled [here](https://build.prestashop-project.org/news/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it.
+Thank you for submitting this issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must follow the template and provide clear and concise steps to reproduce the bug**. 
+Read more about how we expect the issues to be handled [here](https://build.prestashop-project.org/news/2020/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it. 
 
 Thank you
 ```
 
-#### Issue is a functional feature request
+### Issue is a functional feature request
```
Comment (kpodemski) [inline content/project-organization/quality-council/processes/issue-management.md:385]: That part is deprecated, right? somebody from the project should be informed about it, so that we could transfer it to the Discussions.

File: content/project-organization/quality-council/processes/issue-management.md [hunk 10]
```diff
@@ -409,37 +418,35 @@ We only use GitHub issues to discuss about bugs and new features in the PrestaSh
...
-4. Don't hesitate to ping the devs/product/QA on issues where their expertise could help, and to add the corresponding label (`Waiting for PM`, `Waiting for Dev`, etc).
+1. The "NMI" (Need More Info) tag is used at the beginning of the life of the issue, when there are still questions about the issue (quality, completeness, etc). 
+2. "Waiting for Author" is used for conversations with the author and for NMI issues when we answers. The issue-bot will delete the label when the author answers (to help us filtering the issues that needs answers, check the [filter](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3ANMI+-label%3A%22Waiting+for+author%22)).
+3. In case of regression discovery, it is essential to share the regressions on #qa-core (PS Slack) and #dev-core (Public Slack) and to put them in the right project (e.g. if the regression is on PS 1.7.8.5, we put it in the project 1.7.8.5 and in the project [Maxi Kanba](https://github.com/orgs/PrestaShop/projects/7)n, then the PM will prioritize it).
```
Comment (kpodemski) [inline content/project-organization/quality-council/processes/issue-management.md:447]: Maybe we could share it on #project-members for the OS project? Also, correct me if I'm wrong, but I saw earlier that regression is not used from now on, right?

**Review by kpodemski — COMMENTED** [review]
Looks good! I added some questions.

Also, maybe we could add this at the beginning of the document:

```
While the project is transparent, some resources are restricted to PrestaShop SA employees to facilitate task execution.
```

wdyt?

File: content/project-organization/quality-council/processes/issue-management.md (hunk 7 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:239]: Yes, it will be replaced in the next update (all internals documents will be published) by an another member of the QA Council.

File: content/project-organization/quality-council/processes/issue-management.md (hunk 8 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:355]: Fixed

File: content/project-organization/quality-council/processes/issue-management.md (hunk 9 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:385]: I leave it to @Robin-Fischer-PS to decide as lead of the Quality Council.

File: content/project-organization/quality-council/processes/issue-management.md (hunk 10 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:447]: May be could you propose it at the next meeting of the Quality Council. They are always open to external members.

**Comment by Progi1984** [comment]
Ping @Robin-Fischer-PS & @boubkerbribri

File: content/project-organization/quality-council/processes/issue-management.md [hunk 11]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
 
-#### Validating the issue
+### Validating the issue
 
 If the issue is complete and unique, it's time to validate it! For that, after having correctly labelled it, we use the "Issue validated" model. It is important to ping the team best able to answer the author's questions: the devs, the product, the support, the qa...
 
 ## Messages templates
 
-#### Issue not in english
+### Issue not in english
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:263]:
```suggestion
### Issue not in English
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 12]
```diff
@@ -10,107 +10,122 @@ This document details the processes in place for the position of "Issues Manager
 
 ## The Issue Manager job
 
-Your job is to **qualify issues** submitted to the GitHub project. Good issues help us discover bugs and provide ideas to improve PrestaShop and the native modules. These are the three questions you must ask yourself:
+Your job is to **qualify issues** submitted to the GitHub project. Good issues help us discover bugs and provide ideas to improve PrestaShop and the native modules. These are the three questions you must ask yourself:
 
-- **Is the issue valid?** The issue must be either a bug report or a feature request. We only cover the PrestaShop project (Core + native modules).
-- **Has all the needed information been provided?** The issue must be understandable and contain the required information as provided in the issue template, complemented with any other detail needed for reproduction.
-- **Can it be reproduced?** An issue can't be fixed if it can't be reproduced.
+- **Is the issue valid?** The issue must be either a bug report or a request for feature. We only cover the PrestaShop project (Core + native modules).
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:15]:
```suggestion
- **Is the issue valid?** The issue must be a bug report. We only cover the PrestaShop project (Core + native modules). Request for feature should be converted in GitHub Discussions.
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 13]
```diff
@@ -10,107 +10,122 @@ This document details the processes in place for the position of "Issues Manager
...
-   - **`Bug`** – a defect in the software
-   - **`Feature`** – a request for a new feature, a change in an existing functionality
-   - **`Task`** – an issue to track an action that’s not a fix or improve the software like to release a module or network infrastructure change. It’s a task mainly to be done by a maintainer dev.
-   - **`Epic`** - an epic is a group of issues around a bigger topic.
-
-
+    - **`Bug`** – a defect in the software
+    - **`Deprecated - Feature`** – a new feature, a change in an existing functionality (deprecated since April 2023)*
+    - **`Task`** – an issue to track an action that’s not a fix or improve the software like to release a module or network infrastructure changement. It’s a task mainly to be done by the maintainer dev.
+    - **`Epic`** – a epic is a group of issue that are going to
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:35]: Isn't there a stuff missing at the end of the sentence here?

File: content/project-organization/quality-council/processes/issue-management.md [hunk 14]
```diff
@@ -10,107 +10,122 @@ This document details the processes in place for the position of "Issues Manager
...
-   - **`Invalid`** – the issue is incomplete, cannot be reproduced, or its author is unresponsive.
-
-     In addition to this label, additional labels can be added for informative purposes:
-   
-     - **`Can't reproduce`** – the reported bug cannot be reproduced
-     - **`Duplicate`** – the issue has already been reported
-
-
-4. **Severity** *(only for bugs, more details [here][how-issues-are-sorted])*
+    - **`Fixed`** – the issue been resolved
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:43]:
```suggestion
    - **`Fixed`** – the issue has been resolved
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 15]
```diff
@@ -10,107 +10,122 @@ This document details the processes in place for the position of "Issues Manager
...
+2. **`Deprecated - Regression`** – the bug breaks a feature that was working well before (deprecated since August 2023)*
 3. **`Developer Feature`** – technical feature, targeting developers
-4. [Reproduced in] **`<version>`** – the earliest stable version where the bug has been reproduced or the dev branch if it’s a regression
-5. If a known solution exists for a bug that hasn't been fixed/released yet:
+4. **`<branch>`** – where the bug has been reproduced on current branch like `8.1.x` or `develop` branch
+5. Solution available – known solution for a bug that hasn't been fixed/released yet
     - **`PR available`** – a PR fixing the bug is available
     - **`Workaround available`** – a workaround for the bug has been shared
-    - **`Addons module available`** – a module that solves this bug or that provides this feature is available in the marketplace
 6. **`<Category>`** - **`BO`** (Back-office), **`FO`** (Front-office) or  **`WS`** (Webservices) ... - Which part of the software the feature bug/feature is applied
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:66]: Should we mention the "CO" category?

File: content/project-organization/quality-council/processes/issue-management.md [hunk 16]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
-#### Connecting an issue to an EPIC
+### Connecting an issue to an EPIC
 
-When the issue is a **bug that was reproduced** (meaning with the `Verified` label), it MUST in most cases be connected to an `EPIC` if it already exists.
+{{% notice warning %}}
+When the issue is a **bug that was reproduced** (meaning with the `Verified` label), it MUST in most cases be connected to an Epic if it already exists.
+{{% /notice %}}
 
-To do this, perform a search using this filter [EPIC list](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3AEPIC) where all `EPIC` are listed. If there is a related `EPIC`, add the issue to the `EPIC` (by editing the description) and ping the product team.
+To do this, perform a search in the [Bug classification board](https://github.com/PrestaShop/PrestaShop/projects/11) where all EPIC are listed. If there is a related EPIC, add the issue to the EPIC (by editing the description) and ping the product team.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:247]:
```suggestion
To do this, perform a search in the [Bug classification board](https://github.com/PrestaShop/PrestaShop/projects/11) where all Epic are listed. If there is a related EPIC, add the issue to the Epic (by editing the description) and ping the product team.
```

File: content/project-organization/quality-council/processes/issue-management.md (hunk 13 above)
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:35]:
```suggestion
    - **`Epic`** – an Epic is a group of issue that are going to
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 17]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
 
-If no `EPIC` clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the [kanban](https://github.com/PrestaShop/PrestaShop/projects/6) called "PrestaShop Next Major". The product team will regularly review the issues in this kanban and create new `EPIC`s when needed.
+If no Epic clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the kanban called "waiting for classification". The product team will regularly review the issues in this kanban and create new EPICs when needed.
 
-Note that the product team must be able to understand the bug to be able to classify it in an `EPIC`. This means that if the issue is not correctly described nor labelled, or if this is a very tech issue, it **MUST NOT be added** to the kanban.
+Note that the product team must be able to understand the bug to be able to classify it in an EPIC. This means that if the issue is not correctly described nor labelled, or if this is a very tech issue, it MUST NOT be added to the kanban.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:251]:
```suggestion
Note that the product team must be able to understand the bug to be able to classify it in an Epic. This means that if the issue is not correctly described nor labelled, or if this is a very tech issue, it MUST NOT be added to the kanban.
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 18]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
+{{% /notice %}}
 
-To do this, perform a search using this filter [EPIC list](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3AEPIC) where all `EPIC` are listed. If there is a related `EPIC`, add the issue to the `EPIC` (by editing the description) and ping the product team.
+To do this, perform a search in the [Bug classification board](https://github.com/PrestaShop/PrestaShop/projects/11) where all EPIC are listed. If there is a related EPIC, add the issue to the EPIC (by editing the description) and ping the product team.
 
-If no `EPIC` clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the [kanban](https://github.com/PrestaShop/PrestaShop/projects/6) called "PrestaShop Next Major". The product team will regularly review the issues in this kanban and create new `EPIC`s when needed.
+If no Epic clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the kanban called "waiting for classification". The product team will regularly review the issues in this kanban and create new EPICs when needed.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:249]:
```suggestion
If no Epic clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the kanban called "waiting for classification". The product team will regularly review the issues in this kanban and create new Epics when needed.
```

File: content/project-organization/quality-council/processes/issue-management.md (hunk 9 above)
Comment (Robin-Fischer-PS) [reply content/project-organization/quality-council/processes/issue-management.md:385]: This part is indeed Deprecated. The process right now is to convert issue for feature request into a GH Discussion, but we do not have (yet) a template message for that.

File: content/project-organization/quality-council/processes/issue-management.md (hunk 10 above)
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:447]:
```suggestion
3. In case of regression discovery, it is essential to share the regressions on #dev-core (Public Slack) and to put them in the right project (e.g. if the regression is on PS 1.7.8.5, we put it in the project 1.7.8.5 and in the project [Maxi Kanba](https://github.com/orgs/PrestaShop/projects/7)n, then the PM will prioritize it).
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 19]
```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
...
 
 If the issue is reproduced: the bug follows the normal course: go to the next step.
 
-If the issue is not reproduced: **It’s a regression!** Test all the latest patch versions of the current minor version (e.g. 1760, 1761, 1762...) to find out in which version the issue has been introduced. If motivated, look for the PR in question. Then the issue follows the normal course: go to the next step. In addition, it has to be posted on `#qa-core` channel (Corp Slack) and `#dev-core` (Public Slack). The Project must then be labeled by selecting the **label corresponding to the version in which the regression was detected.**
+If the issue is not reproduced: **It’s a regression!** Test all the latest patch versions of the current minor version (e.g. 1760, 1761, 1762...) to find out in which version the issue has been introduced. If motivated, look for the PR in question. Then the issue follows the normal course: go to the next step. In addition, it has to be posted on #qa-core channel (PS Slack) and #dev-core (Public Slack). The Project must then be labeled by selecting the **label corresponding to the version in which the regression was detected.**
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:215]:
```suggestion
If the issue is not reproduced: **It’s a regression!** Test all the latest patch versions of the current minor version (e.g. 1760, 1761, 1762...) to find out in which version the issue has been introduced. If motivated, look for the PR in question. Then the issue follows the normal course: go to the next step. In addition, it has to be posted on #dev-core (Public Slack). The Project must then be labeled by selecting the **label corresponding to the version in which the regression was detected.**
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 20]
```diff
@@ -409,37 +418,35 @@ We only use GitHub issues to discuss about bugs and new features in the PrestaSh
...
     - If you ping someone, ask a direct question! Don't waste their time by making them read the whole thread and guess why did you ping them.
-    - If the issue is too technical to reproduce, ping the Sheriff on Slack (DM) while adding the `TBR` and `Waiting for Dev` labels.
-5. If you don’t know if the issue is `Major` or `Minor`, please post it in `#dev-core` (Public Slack) and ping PM and QA lead, so they can settle the severity. Otherwise, you can check [how issues are sorted][how-issues-are-sorted].
+    - If the issue is too technical to reproduce, ping the Sheriff on Slack (DM) while adding the “TBR” and “Waiting for Dev” labels. ([Sheriff's Calendar](https://www.notion.so/Open-Source-Sheriff-deprecated-be69f240a64e46f892ae75d233cc00b6?pvs=21))
 
-[how-issues-are-sorted]: {{< relref "/get-involved/report-issues/how-issues-are-sorted" >}}
+5. If you don’t know if the issue `Major` or `Minor`, please post it in [#qa-core](https://app.slack.com/client/T029HS3DQ/C89MEQ2CB) and ping PM (@core-pm) and [QA lead](https://www.notion.so/Robin-Fischer-PS-3a6751548f414904beaca65af7adfdb7?pvs=21), so they can settle the severity.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:452]:
```suggestion
5. If you don’t know if the issue `Major` or `Minor`, please post it in [#quality-assurance](https://app.slack.com/client/TTJ6M3GPL/C05M16TEEGG) and ping QA lead (@Robin Fischer), so they can settle the severity.
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 21]
```diff
@@ -409,37 +418,35 @@ We only use GitHub issues to discuss about bugs and new features in the PrestaSh
...
+4. Don't hesitate to ping the devs/product/QA on issues where their expertise could help, and to add the corresponding label ("Waiting for PM", "Waiting for Dev", etc).
     - If you ping someone, ask a direct question! Don't waste their time by making them read the whole thread and guess why did you ping them.
-    - If the issue is too technical to reproduce, ping the Sheriff on Slack (DM) while adding the `TBR` and `Waiting for Dev` labels.
-5. If you don’t know if the issue is `Major` or `Minor`, please post it in `#dev-core` (Public Slack) and ping PM and QA lead, so they can settle the severity. Otherwise, you can check [how issues are sorted][how-issues-are-sorted].
+    - If the issue is too technical to reproduce, ping the Sheriff on Slack (DM) while adding the “TBR” and “Waiting for Dev” labels. ([Sheriff's Calendar](https://www.notion.so/Open-Source-Sheriff-deprecated-be69f240a64e46f892ae75d233cc00b6?pvs=21))
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:450]:
```suggestion
    - If the issue is too technical to reproduce, add the “TBR” and “Waiting for Dev” labels. You should also ping @PrestaShop/committers on the issue.
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 22]
```diff
@@ -409,37 +418,35 @@ We only use GitHub issues to discuss about bugs and new features in the PrestaSh
...
-3. In case of regression discovery, it is essential to share the regressions on #dev-core (Public Slack) and to put them in the right project (e.g. if the regression is on PS 8.0.0, we put it in the project 8.0.x, then the PM will prioritize it).
-4. Don't hesitate to ping the devs/product/QA on issues where their expertise could help, and to add the corresponding label (`Waiting for PM`, `Waiting for Dev`, etc).
+1. The "NMI" (Need More Info) tag is used at the beginning of the life of the issue, when there are still questions about the issue (quality, completeness, etc). 
+2. "Waiting for Author" is used for conversations with the author and for NMI issues when we answers. The issue-bot will delete the label when the author answers (to help us filtering the issues that needs answers, check the [filter](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3ANMI+-label%3A%22Waiting+for+author%22)).
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:446]:
```suggestion
2. "Waiting for Author" is used for conversations with the author and for NMI issues when we answers. The issue-bot will delete the label when the author answer (to help us filtering the issues that needs answers, check the [filter](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3ANMI+-label%3A%22Waiting+for+author%22)).
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 23]
```diff
@@ -261,12 +282,12 @@ Maintenance for this version is now being performed exclusively by volunteers on
...
 ```
 
-#### Incomplete issue
+### Incomplete issue
 
 ```
 Hello,
 
-Could you please provide us with more info?
+Could you please provide us with more info ?
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:290]:
```suggestion
Could you please provide us with more info?
```

File: content/project-organization/quality-council/processes/issue-management.md [hunk 24]
```diff
@@ -344,45 +362,36 @@ You can personalize the message according to your needs, for example:
 For active contributors, it’s not necessary to add this block:
 
 ```
-If this one is important to you [...], the better PrestaShop becomes for everyone.
+If this one is important to you [...] , the better PrestaShop becomes for everyone.
```
Comment (Robin-Fischer-PS) [inline content/project-organization/quality-council/processes/issue-management.md:365]:
```suggestion
If this one is important to you [...], the better PrestaShop becomes for everyone.
```

**Review by Robin-Fischer-PS — CHANGES_REQUESTED** [review]
Some changes and questions, I also removed some internal process (and link to Notion). I might have other suggestion for feature request and regression parts (but I cannot think any more right now).

File: content/project-organization/quality-council/processes/issue-management.md (hunk 13 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:35]: It's the same in Notion.

File: content/project-organization/quality-council/processes/issue-management.md (hunk 15 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/issue-management.md:66]: As you want :)

**Review by Robin-Fischer-PS — APPROVED** [review]
LGTM :)
